src/simdjson_ffi.cpp

```diff
@@ -135,13 +135,23 @@ simdjson_ffi_state *simdjson_ffi_state_new() {
 }
 
 
+// We try to minimize the memory usage for short json string,
```

Comment:
"json string" may cause confusion.

```suggestion
// We try to minimize the memory usage for short json,
```